### PR TITLE
[TASK] Update key generation for Windows

### PR DESCRIPTION
**Pull request by gilbertsoft**

### CODE REVIEW

File: Documentation/Appendix/Windows/SSHKeyWindows.rst [hunk 1]
```diff
@@ -6,47 +6,61 @@
...
-pageant will run as background task and will place a nice little icon into your tray icons.
+Putty provides a key agent named ``Pageant``, which can be added to your
+autostart folder. This allows you to enter your passphrase only once at startup
+and any tools using putty (e.g. git-for-windows, TortoiseGit, etc.) can
+automatically use your private key.
+
+In order to achieve this, create a new shortcut:
+
+* Press :kbd:`Windows` + :kbd:`r`
+* Copy and insert :file:`%APPDATA%\Microsoft\Windows\Start Menu\Programs\Startup`
```
Comment (liayn) [inline Documentation/Appendix/Windows/SSHKeyWindows.rst]: The backslashes are swallowed in the rendering

File: Documentation/Appendix/Windows/SSHKeyWindows.rst [hunk 2]
```diff
@@ -6,47 +6,61 @@
...
+* Right click into the folder and select ``New->Short Cut``
+* Copy and insert %ProgramFiles%\PuTTY\pageant.exe "<full path to your saved
```
Comment (liayn) [inline Documentation/Appendix/Windows/SSHKeyWindows.rst]: same here

File: Documentation/Appendix/Windows/SSHKeyWindows.rst [hunk 3]
```diff
@@ -6,47 +6,61 @@
...
+  private key.ppk>".
+* Click ``Next``. If you get an error the program could not be found, delete
+  the argument with your private key file and continue without an argument
+* Name your new shortcut and click ``Finish``
+* If you had to delete your private key file, check the properties of the newly
```
Comment (liayn) [inline Documentation/Appendix/Windows/SSHKeyWindows.rst]: Maybe rephrase this sentence it is not clear that it does not belong to the setup instructions but rather to a troubleshooting section.

File: Documentation/Appendix/Windows/SSHKeyWindows.rst (hunk 1 above)
Comment (gilbertsoft) [reply Documentation/Appendix/Windows/SSHKeyWindows.rst]: see #149

File: Documentation/Appendix/Windows/SSHKeyWindows.rst (hunk 2 above)
Comment (gilbertsoft) [reply Documentation/Appendix/Windows/SSHKeyWindows.rst]: see #149

File: Documentation/Appendix/Windows/SSHKeyWindows.rst (hunk 3 above)
Comment (gilbertsoft) [reply Documentation/Appendix/Windows/SSHKeyWindows.rst]: see #149

**Comment by gilbertsoft** [comment]
Additional corrections are in #149
